src/app/shared/components/components/edit-product-modal/edit-product-modal.component.scss

```diff
@@ -0,0 +1,95 @@
+.modal {
+  position: fixed;
+  z-index: 2000;
+  padding-top: 100px;
+  left: 0;
+  top: 0;
+  width: 100%;
+  height: 100%;
+  overflow: auto;
+  background-color: rgb(0, 0, 0);
+  background-color: rgba(0, 0, 0, 0.4);
```

Comment:
Repeating code -- you can make a generic modal component, and put the relevant content into `edit`, `remove`, `add` modals